### ISSUE TITLE
Bot keeps crashing after a while (around half an hour i think)

### ISSUE THREAD

**Issue by ExileRL** [MCSulphate/AcceptBot#2]
Hey, just want to firstly say this is an amazing bot :D. It's way better than the one I used a year ago which stopped working.

When someone messages me who is on my friends list already, it sends them my automatic message again and again. Is there any way to fix this? I posted some images of the chat and the modified json file.
![image](https://user-images.githubusercontent.com/59202293/71414920-1bde9580-2651-11ea-8502-3e6bfed311d0.png)
![image](https://user-images.githubusercontent.com/59202293/71414951-42043580-2651-11ea-8d3d-5d0c7d18573e.png)

Any help with this would be awesome and also thanks again for making such an epic bot :D

**Comment by ExileRL** [comment]
[Here's a video which I made to explain the old bot that no longer works.](https://www.youtube.com/watch?v=CiLW9NppBSM&t=717s) Someone made it for me but I don't understand half the stuff since I did very very barebones basic python and not java. That bot only posted the message once, if it helps you could maybe find the part of the code that did that (there's also a google drive link in the description or [here ](https://drive.google.com/drive/folders/1cKeq3H44VIRz35wgdzouSu7EF6R2vQqB?usp=sharing)which has the code to download so you can read it)

**Comment by sulphi-fox** [comment]
Right :) What you need to do is set 'autoMessageEnabled' to 'false'. Auto message means that it will respond to all messages - the 'message' field is probably what you want, this sends a message to the account it accepts when it accepts it, and only once.

**Comment by ExileRL** [comment]
oooh thanks man, im just kinda stupid i guess xD
again thanks for the amazing program this is superb :D

**Comment by ExileRL** [comment]
New issue (which is why i renamed). The bot keeps coming up with this error.
![image](https://user-images.githubusercontent.com/59202293/71421651-1f374880-2674-11ea-9746-906890e2d286.png)

I didnt change any code apart from in the main file. How can I prevent this?

**Comment by sulphi-fox** [comment]
Hmm right.. I guess I didn't cover this when I coded the bot. I'll have a look into this and get back to you on a fix. What's happening here is your logon session (what allows you to use your account) has been  replaced - I'm not sure if this means it expired, but a possible cause is you logged into your account from another location. You are not able to logon from two different places without causing issues, for the most part.

**Comment by ExileRL** [comment]
alright, that makes a bit of sense to me (im half smart half braindead so xD), thanks for taking the time to work it out <3

**Comment by sulphi-fox** [comment]
No problem! Glad that my bot is of use to you :)

**Comment by ExileRL** [comment]
if it helps, while i use the bot i have rocket league open too. i also browse steam discussion pages as i use both rl and the bot (im already signed onto the steam pages from previous instances, so i dont relogin). It seems to crash randomly, so if i listed out the times it crashed:

-1st crash was midway a game (although I've played games before and after that without crashes)
-2nd crash I was not at my pc but it wasnt being used.

Oh and i also am signed in on my phone but I don't use that (apart from to get the auth code to login to the bot)
